lib/src/Uncertainty/Algorithm/MetaModel/MetaModelAlgorithm.cxx

```diff
@@ -43,14 +44,39 @@ MetaModelAlgorithm::MetaModelAlgorithm()
   // Nothing to do
 }
 
+/* Constructor with parameters */
+MetaModelAlgorithm::MetaModelAlgorithm(const Sample & inputSample,
+                                       const Sample & outputSample)
+  : MetaModelAlgorithm(inputSample, outputSample, Dirac(Point(inputSample.getDimension(), 0.0)))
+{
+  // Nothing to do
+}
+
+/* Constructor with parameters */
+MetaModelAlgorithm::MetaModelAlgorithm(const Sample & inputSample,
+                                       const Sample & outputSample,
+                                       const Distribution & distribution,
+                                       const Function & model)
```

Comment:
I don't see the point of keeping a model if we have an output sample.

Reply:
The model accessors are deprecated; we keep them for this release then remove them (and the attributes/ctors) in the new release.